stack/Pulumi.fewgenomes.yaml

```diff
@@ -0,0 +1,8 @@
+encryptionsalt: v1:SPQXlVggjxw=:v1:X1sTpViNuyK+Wcom:3GANI8gZOKtk0gG7BklsyHeNU5uVLw==
+config:
+  datasets:customer_id: C010ys3gt
+  datasets:enable_release: "false"
+  datasets:hail_service_account: fewgenomes-911@hail-295901.iam.gserviceaccount.com
```

Comment:
Are these okay to be public facing? I know they're just service accounts.

Reply:
Thanks for checking! Yes, I don't think somebody else could use this information without having additional permissions.